examples/huggingface/run.py

```diff
@@ -103,13 +103,15 @@ def main(
     **kwargs
 ):
     if nlp_task == "token-classification":
-        # Run Pipeline
         token_classification_config = HuggingfaceParameters(
             label_all_tokens=True,
-            pretrained_model=pretrained_model,
-            epochs=epochs,
-            batch_size=batch_size,
+            pretrained_model=pretrained_model or "distilbert-base-uncased",
+            epochs=epochs or 1,
```

Comment:
`epochs` is an int and will therefore never be optional, the `or` won't be called (unless someone passes 0 but that's just a user error that should crash IMO)